docsteady/vcd.py

```diff
@@ -44,6 +44,22 @@ def extract_fields(self, data):
         return data
 
 
+class Coverage(object):
```

Comment:
First, you don't need to inherit from `object` anymore.  Second, you should look at https://docs.python.org/3/library/dataclasses.html for a simpler way of doing this.  Third, I haven't looked closely, but you may still want the names/labels in a config rather than here; it's a bit strange to be mixing counters and fixed strings in a single class.